http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -764,18 +773,20 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
           config.ingressConfig.release,
           config.ingressConfig.chartName,
           config.ingressConfig.chartVersion,
-          org.broadinstitute.dsp.Values(config.ingressConfig.values.map(_.value).mkString(","))
+          org.broadinstitute.dsp.Values(config.ingressConfig.values.map(_.value).mkString(",")),
+          true
         )
         .run(helmAuthContext)
 
       // Monitor nginx until public IP is accessible
-      loadBalancerIpOpt <- streamFUntilDone(
+      loadBalancerIpOpt <- streamUntilDoneOrTimeout(
```

Comment:
This would now throw a `StreamTimeoutError` I think if the IP does not become available -- could we make it throw `ClusterCreationException` as before? (I think there is some handling attached to these exception types.)

Reply:
that makes sense..after a closer look, `streamFUntilDone` is probably better anyway becuz in this case, "doneness" is whether the returned option becomes `Some`